### PR TITLE
Sager Updates Jan 2022

### PR DESCRIPTION
**Pull request by Kayleen88**

### CODE REVIEW

**Comment by norseman99** [comment]
Hey @Kayleen88,

There are several issues here.

image

As you can see:
* The portrait image doesn't follow the correct size and alignment.
* The perk images are smaller than they should be.
* The perks are missing their names (this goes in randomizer-ui-translator.js file)
* The addons seem more pixelated than they should be. I'm guessing the perk size is incorrect. 
* Killer power image and text is missing.

image

Regardless, you have done a good job. 

Out of curiosity: Do you have a technical background or you did you figure this out for yourself?

I might pick this up and try to fix these issues but I can't promise when. In the meanwhile, feel free to look into them.

You can test the changes locally opening index.html in chrome.

**Comment by Kayleen88** [comment]
Hi,

Thanks for picking up the issues (I don't play DBD so not sure what was missing etc. just added where I saw the most logical stuff).

I am a Systems Analyst / Automation Engineer (doing coding courses after hours) so front-end/images not my strongest side, but will have a look at those this evening.

**Comment by Kayleen88** [comment]
Hi,

Hope you are well.

I have reworked all the images as well as added the data to the UI Translator file. I could only test the portraits by loading the index.html locally - after that it would not work for me anymore :(. 

Not sure if the changes has come through to you - the Pull Request just opens this one.

Keep well
Kayleen

File: script/randomizer-ui-translator.js [hunk 1]
```diff
@@ -70,6 +75,10 @@ const TRANSLATIONS = {
         "CHARACTER_EXECUTIONER": "Pyramid Head (Executioner)",
         "CHARACTER_BLIGHT": "Talbot Grimes (Blight)",
         "CHARACTER_TWINS": "Victor & Charlotte Deshayes",
+        "CHARACTER_ARTIST": "Carmina Mora (Artist)",
+        "CHARACTER_TRICKSTER": "Ji-Woon Hak (Trickster)":,
```
Comment (norseman99) [inline script/randomizer-ui-translator.js:79]: You have a typo at the end of the line. This is probably the reason index.html was no longer working for you.

Should be: `"CHARACTER_TRICKSTER": "Ji-Woon Hak (Trickster)",`

I'll take a look on everything as soon as I can.

Cheers!
Norse

**Comment by norseman99** [comment]
Hey Kayleen,

This has ended up being much more worked and I wanted. 😭 

I ended up only using most of the data and translations files. All images had to be replaced. They were too different from the old ones. It created a really ugly inconsistence in the aesthetic. 

Also fixed several typos in the data file and issues with file names case sensitive. 
 
After several hours working on this I think I manage to resolve most issues. Although, I wouldn't be surprised if more still exist.

You can see all the changes here: https://github.com/norseman99/dbd-randomizer/pull/4

I already did a few fixes I found after. I added credits to your work. Let me know if you want your name linked to somewhere.

**Comment by Kayleen88** [comment]
Hi Norseman,

Thanks for picking up on the typo's (did go through them to double check - but clearly missed them). I feel like a just created more work for you :(

Thank you for finishing it up, I do appreciate it. You did not have to give me credits - this was a little project where I learnt a lot about images and the different files that can be used. 

Keep well
Kayleen

**Comment by norseman99** [comment]
I did it anyway as you have in fact contributed. It's up to you if you want to have your name linked or not.

I hope you learned from the process. I hope to see a new contribution when the next DLC comes out. Hopefully, with less intervention from myself. 😆

**Pull request closed without merging**